### PR TITLE
Improve initial settings

### PR DESCRIPTION
**Pull request by bubanoid**

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -0,0 +1,159 @@
+# Makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    =
+SPHINXBUILD = /data/an.andrusyk/OP_PROJECTS/openregistry.buildout/bin/sphinx-build
```
Comment (leits) [inline docs/Makefile]: @bubanoid /data/an.andrusyk/OP_PROJECTS/

Comment (bubanoid) [reply docs/Makefile]: @Leits Fixed

File: docs/make.bat [hunk 2]
```diff
@@ -0,0 +1,199 @@
+@ECHO OFF
+
+REM Command file for Sphinx documentation
+
+if "%SPHINXBUILD%" == "" (
+	set SPHINXBUILD=sphinx-build
+)
+set BUILDDIR=/data/an.andrusyk/OP_PROJECTS/openregistry.buildout/docs
```
Comment (leits) [inline docs/make.bat]: @bubanoid  /data/an.andrusyk/OP_PROJECTS/

Comment (bubanoid) [reply docs/make.bat]: @Leits Fixed

**Pull request merged**